MyTest/src/test/java/com/test/ui/RestAssuredTest.java

```diff
@@ -0,0 +1,49 @@
+package com.test.ui;
+
+import org.hamcrest.Matchers;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import java.util.List;
+
+import static io.restassured.RestAssured.given;
+import static io.restassured.module.jsv.JsonSchemaValidator.matchesJsonSchemaInClasspath;
+import static pages.ExpectedList.expectedList;
+
+public class RestAssuredTest {
+    private static  final String URL = "http://ergast.com/api/f1/2010/constructors.json";
+    private static final int TOTAL_RECORDS = 12;
+
+    @Test
+    public void isQuantityOfRecordsCorrect(){
+
+        int result = Integer.parseInt(given().when().get(URL).then().extract().path("MRData.total"));
+        Assert.assertTrue(result==TOTAL_RECORDS);
```

Comment:
It will be better to assert it using RestAssured and method body("path to element", Matcher.methodThatEqualResult)